modules/configuration/src/main/java/org/apache/ignite/internal/configuration/asm/InnerNodeAsmGenerator.java

```diff
@@ -514,6 +533,11 @@ private FieldDefinition addInnerNodeField(Field schemaField) {
             throw new IllegalArgumentException("Unsupported field: " + schemaField);
         }
 
+        dieldToDieldDefinitionMap.put(
+                schemaField,
+                innerNodeClassDef.declareField(EnumSet.of(PUBLIC, STATIC, FINAL), fieldName + "FieldDefinition", Field.class)
```

Comment:
`fieldName + "FieldDefinition"` - poor naming. What do you mean by field definition here? It looks more list "schema field". I would also make it upper case, because it's a `static final` field.

Doesn't it break "internal extensions"? They may have repeating properties names, don't we have such tests? Please check